src/astro/databases/base.py

```diff
@@ -250,3 +281,66 @@ def export_table_to_file(
 
         df = self.export_table_to_pandas_dataframe(source_table)
         target_file.create_from_dataframe(df)
+
+    # ---------------------------------------------------------
+    # Schema Management
+    # ---------------------------------------------------------
+
+    def create_schema_if_needed(self, schema):
+        """
+        This function checks if the expected schema exists in the database. If the schema does not exist,
+        it will attempt to create it.
+
+        :param schema: DB Schema - a namespace that contains named objects like (tables, functions, etc)
+        """
+        if not schema:
+            return
```

Comment:
This should never happen though and if it happens we should probably raise an error to avoid: `None`, `""` etc

Reply:
@kaxil this would happen with sqlite

Reply:
Yes but for Sqlite, we will override the `create_schema_if_needed` and return directly in `database/sqlite.py`, we don't need this check here

Reply:
I'm addressing this